tests/data/circuit_config.json

```diff
@@ -15,10 +15,12 @@
         "nodes_file": "$NETWORK_DIR/nodes.h5",
         "populations": {
             "default": {
-                "type": "biophysical"
+                "type": "biophysical",
+                "spatial_segment_index_dir": "path/to/node/dir"
             },
             "default2": {
-                "type": "biophysical"
+                "type": "biophysical",
+                "spatial_segment_index_dir": "path/to/node/dir2"
```

Comment:
It could be useful to test also a node/edge population without the spatial index, because it's optional.

Reply:
I added such a think, not sure if it is what you had in mind :slightly_smiling_face:

Reply:
I was thinking to keep 1 config file, where the index was defined for one population, and not defined for the other population, but using 2 config files works as well.

Reply:
I was thinking to keep 1 config file, where the index was defined for one population, and not defined for the other population, but using 2 config files works as well.

Reply:
that is better :+1: